src/index.js

```diff
@@ -1,22 +1,29 @@
 import 'assets/styles/main.scss';
 import VideoPlayer from './video-player';
 import { assign } from 'utils/assign';
-import { omit, pick } from './utils/object';
+import { omit, pick, convertKeysToSnakeCase } from './utils/object';
 import { CLOUDINARY_CONFIG_PARAM } from './video-player.const';
 
 if (window.cloudinary && window.cloudinary.Cloudinary) {
-  console.warn('For version 1.9.0 and above, cloudinary-core is not needed for using the Cloudinary Video Player');
+  console.warn(
+    'For version 1.9.0 and above, cloudinary-core is not needed for using the Cloudinary Video Player'
+  );
 }
 
-const getConfig = (playerOptions = {}, cloudinaryConfig) => assign(
-  omit(playerOptions, CLOUDINARY_CONFIG_PARAM),
-  { cloudinaryConfig: cloudinaryConfig || pick(playerOptions, CLOUDINARY_CONFIG_PARAM) });
+const getConfig = (playerOptions = {}, cloudinaryConfig) =>
+  // pick cld-configurations and assign them to cloudinaryConfig
+  assign(omit(playerOptions, CLOUDINARY_CONFIG_PARAM), {
+    cloudinaryConfig:
+      cloudinaryConfig || convertKeysToSnakeCase(pick(playerOptions, CLOUDINARY_CONFIG_PARAM))
```

Comment:
I actually like that, but as it is, the `omit` will not work as intended. this will cause the snakeCase `cloudName` to be passed to the player `source()` method. but I'm looking into that still...

Reply:
looking on this again I'm not sure we even need to `omit` anything, since this goes through `extractOptions()` in `video-player`'s constructor

Reply:
Done ✅